doc/source/metadata/Reference.rst

```diff
@@ -0,0 +1,577 @@
+of the data (typically an NWMP model)
+when no significant post-processing has been applied.
+However, here significant adjustment of the data has occurred
+or a number input sources have been blended,
+it **should** be set to ``IMPROVER``.
+Often, careful consideration of when it is appropriate to set this
+to reference ``IMPROVER`` is required to avoid the metadata being misleading.
+It is probably not worth including a version of the IMPROVER software,
+unless this can be reliably supplied.
+
+spot_index
+**********
+
+This is an IMPROVER-specific dimension is used as an increasing integer value
+index for sites.
+
+ssp\__
+******
+
+This is intended to indicate a SPP (statistical post-processing)
+namespace.
+It prefixes atributes to show that they are separate from the 
+`CF Metadata Conventions`_ attributes.
+
+ssp__relative_to_threshold
+**************************
+
+This is an IMPROVER-specific varaible attribute
+indicates the nature of the threshold inequality for a probability
+and takes one of the four values:
+
+* ``greater_than`` 
+* ``greater_than_or_equal_to``
+* ``less_than`` 
+* ``less_than_or_equal_to``
+
+standard_name
+*************
+
+This CF attribute provides a descriptive name,
+from the governed `CF Standard Name`_ list.
+If no `standard_name`` exists for the quantity, 
+a ``long_name`` must be used.
+
+A ``standard_name`` or ``long_name`` **must** be present. 
+
+string5 / string8
+*****************
+
+These IMPROVER-specific arbitary constants are used to dimension
+the character length of the string variable holding
+zero padded WMO identifiers and Met Office identifiers, respectively.
+
+threshold
+*********
+
+This is an IMPROVER-specific coordinate variable that holds
+the set of values of the variable of interest for which the
+probability values are generated.
+It has a ``long_name`` attribute set to ``threshold``.
+
+This **must** be present for probability variables.
+
+time
+****
+
+This CF Variable provides the time at which the parameter value is valid,
+and has a ``standard_name`` attribute set to ``time``.
+This is an 64-bit integer in ``units`` of ``seconds since 1970-01-01 00:00:00``
+
+This **must** be present.
+If any statistical processing over time has been applied
+(e.g. accumulation, maxiumum, etc),
+there **must** also be ``time_bnds`` variable
+providing the time bounds over which ``cell_methods`` are applied.
+``time_bnds`` has no attributes as it is tied to the main time variable.
+
+title
+*****
+
+This netCDF global attribute provides a succinct description
+of what is in the file and should be something that could be used on a plot
+to help describe the data. 
+
+This **must** be present, but there is no generally prescribed form
+that is must take.
+
+units
+*****
+
+This netCDF variable attribute provides the units of measurement for the quantity.
+in a string form recognised by the Unidata's `UDUNITS package`_
+
+This **must** be present,and for IMPROVER this **must** be SI units,
```

Comment:
```suggestion
This **must** be present and for IMPROVER this **must** be SI units,
```
I don't think that this comma is necessary

Reply:
Probably note - removed.